modules/nw-dpu-creating-a-sfc.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * networking/networking_operators/nw-dpu-running-workloads.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-a-sfc-dpu_{context}"]
```

Comment:
The ID tag is different to the file name. Ideally keep them the same to avoid link rendering issues.